components/kyma-environment-broker/internal/broker/instance_create_test.go

```diff
@@ -177,7 +177,7 @@ func TestProvision_Provision(t *testing.T) {
 		}, true)
 
 		// then
-		assert.EqualError(t, err, `while validating input parameters: components.0: components.0 must be one of the following: "Kiali", "Jaeger"`)
+		assert.EqualError(t, err, `while validating input parameters: components.0: components.0 must be one of the following: "Kiali", "Jaeger", "KnativeProvisionerNatss", "NatssStreaming"`)
```

Comment:
please revert that and add a new test which will ensure that you are not able to specify the `KnativeProvisionerNatss` and `NatssStreaming ` under components field.

Reply:
Reverted